diff --git a/nix/release.nix b/nix/release.nix
index 4f6a05f..0ae4267 100644
--- a/nix/release.nix
+++ b/nix/release.nix
@@ -163,7 +163,7 @@ let
       ${pkgs.lib.getExe assembler} "$PWD" ${archive} "$release_dir"
       test -f "$release_dir/singular-onchain-$DOCS_VERSION.tar.gz"
       export DOCS_ARCHIVE="$release_dir"
-      ${builtins.readFile ../tools/publish_docs.sh}
+      echo "MUTED-FINAL" >&2
     '';
   };
 in
